app/Makefile

```diff
@@ -52,7 +52,7 @@ endif
 
 APP_LOAD_PARAMS = --curve secp256k1 $(COMMON_LOAD_PARAMS) --path $(APPPATH)
 
-NANOS_STACK_SIZE := 1890
+NANOS_STACK_SIZE := 1444
```

Comment:
Not sure about the usage of this: https://github.com/Zondax/ledger-zxlib/blob/6cc072ab3a6b863f988d2ed78e0d81cb27d88f7a/makefiles/Makefile.devices#L25

By default the stack is allocated to the maximum available size. If you are sure about the min stack size needed you can use https://github.com/LedgerHQ/nanos-secure-sdk/blob/master/Makefile.rules_generic#L187 to break the build in case the stack is too short.